libcxx/include/__configuration/abi.h

```diff
@@ -24,8 +24,6 @@
 #  define _LIBCPP_ABI_ALTERNATE_STRING_LAYOUT
 // Fix deque iterator type in order to support incomplete types.
 #  define _LIBCPP_ABI_INCOMPLETE_TYPES_IN_DEQUE
-// Fix undefined behavior in how std::list stores its linked nodes.
-#  define _LIBCPP_ABI_LIST_REMOVE_NODE_POINTER_UB
```

Comment:
**If** we make this ABI break unconditionally for `std::list`, I would also do it for the other container types at the same time, since it's basically the same thing.